README.md

```diff
@@ -295,22 +306,16 @@ The Client ID and the Client Secret will be needed for the basic script you will
 
 ### Flexy OCS Configuration File Changes
 1. Add OCS information to the ConnectorConfig.json file.
-   1. Replace the xxx below with the namespace.
-   1. Replace the yyy below with the tenant ID.
-   1. Copy these into the ServerConfig portion of the json file.
-      ```json
-      "Namespace":"xxx",
-      "TenantId":"yyy"
-      ```
-   1. Make sure that the JSON formatting is valid. Below is an example of a properly formatted configuration file.
+   1. Replace the {tenantId} below with the tenant ID of the OCS account.
+   2. Replace the{namespace} below with the namespace of the OCS account.
```

Comment:
space after the?

Reply:
This space is still there.

Reply:
Oh, I thought you meant before the json and after the '?'. will change

Reply:
```suggestion
   2. Replace the {namespace} below with the namespace of the OCS account.
```